telegram/_files/file.py

```diff
@@ -207,7 +207,7 @@ async def download_to_memory(
         filename.write_bytes(buf)
         return filename
 
-    async def download_to_object(
+    async def download_to_memory(
         self,
         out: BinaryIO,
         read_timeout: ODVInput[float] = DEFAULT_NONE,
```

Comment:
btw shouldn't these args be kw-only? Or should we do it in another pr later

Reply:
oh, right. no, I'll fix it. Don't we test for that anywhere? 🤔

Reply:
I think the test is only for bot